pages/jobs.tsx

```diff
@@ -19,7 +20,7 @@ type JobEntryProps = {
 };
 
 const JobEntry = ({ job }: JobEntryProps) => (
-    <Link href={'/jobs/' + job.slug}>
+    <Link href={urls.jobs + job.slug}>
```

Comment:
**Suggestion**
Could we use url objects here: https://nextjs.org/docs/api-reference/next/link#with-url-object 
This might require a new urls key value such as `jobEntry: '/jobs/[slug]/',`

Reply:
That's cool, I'll give this a go 👍